src/GafferImageUI/ImageView.cpp

```diff
@@ -2040,6 +2056,21 @@ void ImageView::preRender()
 		m_imageGadgets[1]->setWipeEnabled( false );
 	}
 
+	if( m_matchDisplayWindows )
```

Comment:
Would dropping `m_matchDisplayWindows` and reading the plug directly here be prohibitively expensive? My hope would be not...